### PR TITLE
feat(BUX-597): send rawtx to arc

### PR DESCRIPTION
**Pull request by arkadiuszos4chain**
# Pull Request Checklist

- [x] 📖 I created my PR using provided  : [CODE_STANDARDS](https://github.com/bitcoin-sv/go-broadcast-client/blob/main/.github/CODE_STANDARDS.md)
- [x] 📖 I have read the short Code of Conduct: [CODE_OF_CONDUCT](https://github.com/bitcoin-sv/go-broadcast-client/blob/main/.github/CODE_OF_CONDUCT.md)
- [x] 🏠 I tested my changes locally.
- [x] ✅ I have provided tests for my changes.
- [x] 📝 I have used conventional commits.
- [x] 📗 I have updated any related documentation.
- [x] 💾 PR was issued based on the Github or Jira issue.

### CODE REVIEW

**Review by chris-4chain — DISMISSED** [review]

**Review by chris-4chain — APPROVED** [review]

**Comment by kuba-4chain** [comment]
Testing...

File: broadcast/internal/arc/arc_submit_tx.go [hunk 1]
```diff
@@ -31,7 +31,8 @@ func (a *ArcClient) SubmitTransaction(ctx context.Context, tx *broadcast.Transac
 		opt(options)
 	}
 
-	result, err := submitTransaction(ctx, a, tx, options)
+	sumbiter := getSubmiter(options)
```
Comment (Nazarii-4chain) [inline broadcast/internal/arc/arc_submit_tx.go:34]: Typo here

File: broadcast/internal/arc/arc_submit_tx.go [hunk 2]
```diff
@@ -64,7 +65,8 @@ func (a *ArcClient) SubmitBatchTransactions(ctx context.Context, txs []*broadcas
 		opt(options)
 	}
 
-	result, err := submitBatchTransactions(ctx, a, txs, options)
+	sumbiter := getSubmiter(options)
```
Comment (Nazarii-4chain) [inline broadcast/internal/arc/arc_submit_tx.go:68]: Typo here

File: broadcast/internal/arc/arc_submit_tx.go [hunk 3]
```diff
@@ -82,11 +84,137 @@ func (a *ArcClient) SubmitBatchTransactions(ctx context.Context, txs []*broadcas
...
+type submitFormatStrategy interface {
+	submit(ctx context.Context, arc *ArcClient, tx *broadcast.Transaction, opts *broadcast.TransactionOpts) (*broadcast.SubmittedTx, error)
+	batchSubmit(ctx context.Context, arc *ArcClient, txs []*broadcast.Transaction, opts *broadcast.TransactionOpts) ([]*broadcast.SubmittedTx, error)
+}
+
+func getSubmiter(opts *broadcast.TransactionOpts) submitFormatStrategy {
+	switch opts.TransactionFormat {
+	case broadcast.RawTxFormat:
+		return &rawtxSubmitStrategy{}
+	case broadcast.EfFormat:
```
Comment (Nazarii-4chain) [inline broadcast/internal/arc/arc_submit_tx.go:96]: We can remove this case as default case is efSubmitStrategy

**Pull request closed without merging**